target_chains/solana/sdk/js/solana_utils/src/transaction.ts

```diff
@@ -419,7 +419,7 @@ export async function sendTransactions(
     signers?: Signer[] | undefined;
   }[],
   connection: Connection,
-  wallet: Wallet,
```

Comment:
will this break things that use sendTransactions in a non-browser context?

Reply:
I highly doubt it. AnchorWallet is a subset of NodeWallet.